README.md

```diff
@@ -1,4 +1,5 @@
 [![Build Status](https://travis-ci.org/Materials-Consortia/optimade-python-tools.svg?branch=master)](https://travis-ci.org/Materials-Consortia/optimade-python-tools)
+[![Heroku](https://heroku-badge.herokuapp.com/?app=optimade&root=optimade/info)](https://optimade.herokuapp.com/optimade)
```

Comment:
If this gets merged before #68, could you please make sure this gets added to that PR?